web-client/src/app/state/session-xrpl.service.ts

```diff
@@ -552,7 +552,7 @@ export class SessionXrplService {
           })
           .then(({ isConfirmed }) => {
             if (isConfirmed) {
-              this.navCtrl.navigateRoot('/');
+              this.navCtrl.back();
```

Comment:
This shouldn't be done in the service, it should be handled on the page. that way you have different behaviours instead of this one back navigation